master/custom/factories.py

```diff
@@ -623,8 +615,8 @@ def setup(self, parallel, branch, test_with_PTY=False, **kwargs):
         self.addStep(
             SetPropertyFromCommand(
                 name="Gather build triple from worker",
-                description="Get the build triple from make",
-                command=["make", "-v"],
+                description="Get the build triple config.guess",
+                command=["./config.guess"],
                 extract_fn=extract_build_triple,
```

Comment:
```suggestion
                command="./config.guess",
                property="build_triple",
```

According to the docs if you use the simpler command interface it already strips stdout, so this should be even cleaner.